client/src/app/management/components/committee-list/committee-list.component.ts

```diff
@@ -59,6 +63,23 @@ export class CommitteeListComponent extends BaseListViewComponent<ViewCommittee>
         this.router.navigate(['create'], { relativeTo: this.route });
     }
 
+    public async forwardToCommittees(): Promise<void> {
+        const content = this.translate.instant(
+            'This will add or remove the following groups for all selected participants:'
+        );
+        const ADD = _('Forward to committee(s)');
+        const REMOVE = _('Unforward to committee(s)');
+        const choices = [ADD, REMOVE];
+        const selectedChoice = await this.choiceService.open(content, this.repo.getViewModelList(), true, choices);
```

Comment:
story for another day, but I suppose we need to make the choice service prettier. Thats some weird buttons.

![image](https://user-images.githubusercontent.com/10233032/115019116-041bc800-9eb9-11eb-9352-e00e6fdb0815.png)